torch/lib/TH/THGeneral.c

```diff
@@ -359,25 +359,27 @@ TH_API void THInferNumThreads(void)
 #endif
 }
 
-TH_API THDescBuff _THSizeDesc(const long *size, const long ndim) {
-  const int L = TH_DESC_BUFF_LEN;
+TH_API THDescBuff _THSizeDesc(const uint8_t *size, const uint8_t ndim) {
```

Comment:
Do you actually want to reduce it to `uint8_t`? Looks like you lose some bits here?